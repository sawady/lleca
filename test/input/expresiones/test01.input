1000 mod 3 + 2000 mod 5 + 3000 mod 7
