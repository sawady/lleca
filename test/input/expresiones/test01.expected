add(
  add(
    mod(
      1000,
      3
    ),
    mod(
      2000,
      5
    )
  ),
  mod(
    3000,
    7
  )
)
